src/components/plot.jsx

```diff
@@ -1,22 +1,44 @@
-import {CartesianGrid, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis} from "recharts"
+import {CartesianGrid, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis, Label} from "recharts"
 import React from "react";
 import PropTypes from 'prop-types';
+import {AxisSide} from "../chart/axisSide";
 
 function Plot({key, chart, width, aspect}) {
+
+    const leftAxis = chart.metadata.leftAxis;
+    const rightAxis = chart.metadata.rightAxis;
+    const leftLabel = `${chart.metadata.yLeftLabel} [${leftAxis.unit}]`
+    const rightLabel = `${chart.metadata.yRightLabel} [${rightAxis.unit}]`
+
     return (
         <div style={{display: 'flex', justifyContent: 'center', alignItems: 'center'}}>
             <ResponsiveContainer width={width} aspect={aspect}>
                 <LineChart data={chart.data}>
-                    <YAxis/>
+                    <YAxis yAxisId={AxisSide.LEFT.value}
+                           orientation={AxisSide.LEFT.value}>
+                        <Label value={leftLabel} angle={-90}/>
+                    </YAxis>
+                    <YAxis yAxisId={AxisSide.RIGHT.value}
+                           orientation={AxisSide.RIGHT.value}>
+                        <Label value={rightLabel} angle={-90}/>
+                    </YAxis>
                     <CartesianGrid stroke="#eee" strokeDasharray="5 5"/>
                     <XAxis dataKey={"timestamp"} hide={false}/>
-                    <YAxis/>
                     {
                         Object.entries(chart.dataSeries).map(dataSeries => {
-                            return (<Line key={dataSeries[0]}
+                            const seriesName = dataSeries[0];
+                            const seriesInfo = dataSeries[1];
+                            const yAxisSide = seriesInfo.yAxisSide;
+                            const axis = yAxisSide.value === AxisSide.LEFT.value ? leftAxis : rightAxis;
```

Comment:
I again propose splityibg the condidon operator into 3 lines

Reply:
Again that's what ternary operator is for - to inline if statements